### PR TITLE
consolidate all the PR news

### PR DESCRIPTION
**Pull request by ngranda**
Removes the PR specific news files and adds their content into one news file. 
Authors of those PRs that were not in the Authors file were added

### CODE REVIEW

**Comment by ngranda** [comment]
Hoping to start a conversation about a DAGMC release with this PR

**Comment by gonuke** [comment]
It makes sense to me to keep the individual news files, even as their content is consolidated into a main news file.

Alternatively, each of the entries in the consolidated file could indicate which PR it came from.

E.g.
```
* something new was added or a bug fixed (PR#987)
```

**Comment by gonuke** [comment]
(Unless someone has an argument against this strategy?)

**Comment by bam241** [comment]
thx @ngranda 
merging

**Comment by ljacobson64** [comment]
So what is the new practice now? Continue to add PR-XXXX.rst files for every PR, and every time we do a release we delete all the news files and combine them into a single file? Perhaps the combined news file should have the version number in its filename?

File: AUTHORS.rst [hunk 1]
```diff
@@ -4,6 +4,10 @@ Authors
 The DAGMC project began as the CAD-MCNPX project in 2003, and since then has
 included contributions from the following:
 
+* Paul Romano
+* Jonathan Shimwell
+* Chelsea D'Angelo
+* Xiaokang Zhang
```
Comment (ljacobson64) [inline AUTHORS.rst:10]: It seems like the rest of the names are in alphabetical order. I feel we should continue that practice

**Pull request merged**